test/k8sT/manifests/1.7/knp-default-deny-ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.k8s.io/v1
+apiVersion: extensions/v1beta1
```

Comment:
Aren't we removing support for `v1beta1`? Why are we changing the API version here?

Reply:
No, we are not removing the support for v1beta1.

```
Since k8s 1.7 provides an automatic translation from KNP
extensions/v1beta1 to networking.k8s.io/v1 we can remove duplicate code
that was translating KNP extensions/v1beta1 to CNP.
```